### PR TITLE
Synchronize GitLab with GitHub

### PR DESCRIPTION
**Pull request by crnh**
## Proposed change

- Pull new analyses and examples by @LucVV to GitHub
- Fix some problems causing the new analysis format not to work on Python <= 3.10

## Type of change

- [x] Example (a notebook demonstrating how to use ZOSPy for a specific application)
- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New analysis (a wrapper around an OpticStudio analysis)
- [ ] New feature (other than an analysis)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests
- [ ] Documentation (improvement of either the docstrings or the documentation website)

# Additional information

- Python version: 3.9-3.12
- OpticStudio version: 24R1.3

## Checklist

- [x] I have followed the [contribution guidelines][contribution-guidelines]
- [x] The code has been linted, formatted and tested locally using tox.
- [x] Local tests pass. **Please fix any problems before opening a PR. If this is not possible, specify what doesn't work and why you can't fix it.**
- [ ] I added new tests for any features contributed, or updated existing tests.
- [x] I updated CHANGELOG.md with my changes (except for refactorings and changes in the documentation).

If you contributed an analysis:

- [x] I did not use `AttrDict`s for the analysis result data (please use dataclasses instead).

If you contributed an example:

- [x] I contributed my example as a Jupyter notebook.
    

[contribution-guidelines]: https://github.com/MREYE-LUMC/ZOSPy/blob/main/CONTRIBUTING.md
[unittest-instructions]: https://github.com/MREYE-LUMC/ZOSPy/blob/main/tests/README.md
[numpydoc]: https://numpydoc.readthedocs.io/en/latest/format.html

### CODE REVIEW

File: tests/data/reference/24.1.3-test_extendedscene.py-test_geometric_image_analysis_returns_correct_result[CrossX-0-1-100000].ZDA
Comment (crnh) [inline tests/data/reference/24.1.3-test_extendedscene.py-test_geometric_image_analysis_returns_correct_result[CrossX-0-1-100000].ZDA:1]: Please remove all ZDA files.

File: examples/Modelling of a Shack-Hartmann Sensor for eye aberration evaluation/2_Fitting_reverse_gradient_3.ipynb
Comment (jwmbeenakker) [inline examples/Modelling of a Shack-Hartmann Sensor for eye aberration evaluation/2_Fitting_reverse_gradient_3.ipynb:1]: please fix the name-error

File: examples/Modelling of a Shack-Hartmann Sensor for eye aberration evaluation/1_Model_SH_original.ipynb
Comment (jwmbeenakker) [inline examples/Modelling of a Shack-Hartmann Sensor for eye aberration evaluation/1_Model_SH_original.ipynb:1]: This complete example is not showing up in readthedocs

Comment (jwmbeenakker) [inline examples/Modelling of a Shack-Hartmann Sensor for eye aberration evaluation/1_Model_SH_original.ipynb:1]: all three notebooks would benefit from plotting the resulting optical setup using the new plotting functionality.

**Review by jwmbeenakker — CHANGES_REQUESTED** [review]

**Review by jwmbeenakker — APPROVED** [review]

**Pull request merged**